feat(changelog): object-storage-added-our-amsterdam-region-is-now-multi- 2023-10-23 (#2305)
scaleway/docs-content: changelog/october2023/2023-10-23-object-storage-added-our-amsterdam-region-is-now-multi-.mdx

---

> title: Our Amsterdam Region is now multi-AZ by design for hot storage!

Reviewer: ```suggestion
title: Our Amsterdam region is now multi-AZ by design for hot storage!
```

status: added
author:
    fullname: 'Join the #object-storage channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2023-10-23
category: storage
product: object-storage
---

> The Multi-AZ Standard class is now available in Amsterdam Region, ensuring the resiliency of your data across 3 Availability Zones (AZ) for all new objects uploaded in Standard class from now on. 

Reviewer 2: ```suggestion
The multi-AZ Standard class is now available in Amsterdam Region, ensuring the resiliency of your data across 3 Availability Zones (AZ) for all new objects uploaded in Standard class from now on. 
```

> The Multi-AZ Standard class is now available in Amsterdam Region, ensuring the resiliency of your data across 3 Availability Zones (AZ) for all new objects uploaded in Standard class from now on. 

Reviewer: ```suggestion
The multi-AZ Standard class is now available in Amsterdam region, ensuring the resiliency of your data across 3 Availability Zones (AZ) for all new objects uploaded in Standard class from now on. 
```

> The multi-AZ Standard class is now available in Amsterdam region, ensuring the resiliency of your data across 3 Availability Zones (AZ) for all new objects uploaded in Standard class from now on. 

Reviewer: ```suggestion
The multi-AZ Standard class is now available in the Amsterdam region, ensuring the resiliency of your data across 3 Availability Zones (AZ) for all new objects uploaded in Standard class from now on. 
```

> This is our second Region to support Multi-AZ by design and to offer 99,9% availability SLA, after Paris Region. Please check out our FAQ for more information https://www.scaleway.com/en/docs/faq/new-object-storage-classes/

Reviewer 2: ```suggestion
This is our second Region to support multi-AZ by design and to offer 99,9% availability SLA, after Paris Region. Please check out our FAQ for more information https://www.scaleway.com/en/docs/faq/new-object-storage-classes/
```

> This is our second Region to support Multi-AZ by design and to offer 99,9% availability SLA, after Paris Region. Please check out our FAQ for more information https://www.scaleway.com/en/docs/faq/new-object-storage-classes/

Reviewer: ```suggestion
This is our second region to support multi-AZ by design and to offer 99,9% of availability SLA, after the Paris region. Check out our [FAQ](/faq/new-object-storage-classes/) for more information.
```
